dist/android/styles/qgds.xml

```diff
@@ -20,6 +20,27 @@
   <color name="qgdsColorNeutralLightestOpacityLow">rgba(245, 245, 245, 0.1)</color>
   <color name="qgdsColorNeutralWhiteDefault">#ffffff</color>
   <color name="qgdsColorNeutralWhiteOpacityLow">rgba(255, 255, 255, 0.1)</color>
+  <string name="qgdsFontSizeMapMobileXs">12.8</string>
+  <string name="qgdsFontSizeMapMobileSm">16</string>
+  <string name="qgdsFontSizeMapMobileMd">16</string>
+  <string name="qgdsFontSizeMapMobileLg">19.2</string>
+  <string name="qgdsFontSizeMapMobileXl">23.04</string>
+  <string name="qgdsFontSizeMapMobileXxl">27.648</string>
+  <string name="qgdsFontSizeMapMobileXxxl">33.1776</string>
+  <string name="qgdsFontSizeMapMobileXxxxl">39.8131</string>
+  <string name="qgdsFontSizeMapDesktopXs">12.8</string>
+  <string name="qgdsFontSizeMapDesktopSm">16</string>
+  <string name="qgdsFontSizeMapDesktopMd">19.2</string>
+  <string name="qgdsFontSizeMapDesktopLg">23.04</string>
+  <string name="qgdsFontSizeMapDesktopXl">27.648</string>
+  <string name="qgdsFontSizeMapDesktopXxl">33.1776</string>
+  <string name="qgdsFontSizeMapDesktopXxxl">39.8131</string>
+  <string name="qgdsFontSizeMapDesktopXxxxl">47.7757</string>
+  <string name="qgdsMediaQuerySm">400px</string>
```

Comment:
why mix numbers and px's

Reply:
Vanilla requires the value as Number: 

$QLD-fontsize-map: (
    xs:   14,
    sm:   16,
    md:   20,
    lg:   24,
    xl:   32,
    xxl:  40,
    xxxl: 52
) !default;